x-pack/test/api_integration/apis/uptime/rest/certs.ts

```diff
@@ -67,19 +77,26 @@ export default function ({ getService }: FtrProviderContext) {
       });
 
       it('retrieves expected cert data', async () => {
-        const apiResponse = await supertest.get(API_URLS.CERTS);
```

Comment:
I think you can delete this key from `x-pack/plugins/uptime/common/constants/rest_api.ts` now.